### PR TITLE
added search feature

### PR DESCRIPTION
**Pull request by Anish-JS**
Please review the search filter on the navbar

### CODE REVIEW

**Comment by samflab** [comment]
i checked the deployed code, when I click on the search button the product page opens and then you can search and get the result but when I click on the hamburger icon it takes me back to the home page and the search bar is still there but I can't search and see the results.
here's
docs

I think you are toggling the search bar, instead what you can do is, only show the search on the product / catalogue page and on the hamburger icon you can show a side nav with the remaining nav-links like Home, Catalogue, etc. And if the user is on any other page the search bar shouldn't be visible.  and hide the hamburger icon on the other pages since the nav-links will be visible.

File: src/App.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { ratingFilter } from "./utils/productFilters/rating";
 import { priceRangeFilter } from "./utils/productFilters/priceRangeFilter";
 import { fastDeliveryFilter } from "./utils/productFilters/fastDeliveryFilter";
 import { outOfStockFilter } from "./utils/productFilters/outOfStockFilter";
+import { searchFilter } from "./utils/productFilters/searchFilter";
```
Comment (samflab) [inline src/App.js:13]: you can use barrel import, it will reduce the lines of code

**Review by samflab — APPROVED** [review]
overall the code is good and easy to understand. just make those UI changes then you are good to go. :D

**Review by Pratik1005 — APPROVED** [review]
Search functionality is working, other than the search toggle issue rest looks good

File: src/App.js (hunk 1 above)
Comment (Anish-JS) [reply src/App.js:13]: done all the changes. Thank you

**Comment by Anish-JS** [comment]
> Search functionality is working, other than the search toggle issue rest looks good

done the changes. Thanks

**Pull request merged**